cime_config/tests.py

```diff
@@ -842,6 +842,7 @@
         "time"  : "03:00:00",
         "tests" : (
             "SMS_D_Lm2.ne4pg2_oQU480.F2010-EAMxx-MAM4xx-MPASSI",
+            "ERS_Ld80.ne4pg2_ne4pg2.F2010-EAMxx-MAM4xx",
```

Comment:
I don't see this testsuite ran by any of our jenkins jobs (or included by a testsuite ran by our jenkins jobs). Are you planning on modify one of our scripts in the [test scripts](https://github.com/E3SM-Project/e3sm_test_scripts) repo?

Reply:
I saw @singhbalwinder make some PRs for E3SM_test_scripts that add mam4xx testing. @singhbalwinder , did you ask @wadeburgess to make a new jenkins job for these?

Reply:
`e3sm_eamxx_mam4xx_long_runtime` test suite is only run on Compy via this script:https://github.com/E3SM-Project/E3SM_test_scripts/blob/master/jenkins/compy_long_runtime.sh. They appear on the dashboard: https://my.cdash.org/builds/3060553

@jgfouca: The new scripts in the `E3SM_test_scripts` repo are for debug tests only. Once we merge those, I will ask @wadeburgess to add them to Jenkins jobs.